src/main.rs

```diff
@@ -236,10 +239,13 @@ async fn run(miniserve_config: MiniserveConfig) -> Result<(), ContextualError> {
 
     let srv = srv.shutdown_timeout(0).run();
 
+    println!("Bound to {}", display_sockets.join(", "));
+
+    println!("Serving path {}", Color::Yellow.paint(path_string).bold());
+
     println!(
-        "Serving path {path} at:\n\t{addresses}\n",
-        path = Color::Yellow.paint(path_string).bold(),
-        addresses = addresses,
+        "Availabe at (non-exhaustive list):\n    {}\n",
+        display_urls.join("\n    "),
```

Comment:
I wounder if there is a better way to encode spaces?

Reply:
Eh, don't sweat it. We don't need to golf _everything_. :P